utils/apiLog.js

```diff
@@ -81,8 +87,9 @@ function combineAndLog(resultObj, logObject, retval) {
  * @param {Object} req - the request object
  * @param {Object} resultObj - Object with the rest of the fields to print
  * @param {Object or Array} retval - the returned object
+ * @param {Integer} recordCount - a custom recordCount
```

Comment:
pls. rename this to "recordCountOverride" and call it out as optional in the field description in the @param comments